src/modules/session-storage/utils.ts

```diff
@@ -0,0 +1,25 @@
+import type { Chain, Hex } from "viem"
+import { SessionFileStorage, SessionLocalStorage } from "../.."
+import type { BiconomySmartAccountV2, SmartAccountSigner } from "../../account"
+import type { ISessionStorage } from "../interfaces/ISessionStorage"
+import { supportsLocalStorage } from "./SessionLocalStorage"
+
+export const createAndStoreNewSessionKey = async (
+  smartAccount: BiconomySmartAccountV2,
+  chain: Chain,
+  _sessionStorageClient?: ISessionStorage
```

Comment:
why _ for this particular var?

Reply:
Just to avoid a namespace conflict inside the method